.env.example

```diff
@@ -1 +0,0 @@
-DATABASE_URL="postgres://user:password@localhost:5432/DB_DEVELOPMENT_DOCKER"
```

Comment:
this would remove the env.example file from the main repo. can you undo this change?

will update the README, probably there was some confusion with the renaming